README.md

```diff
@@ -0,0 +1,49 @@
+# Afkanerd-OpenWebSockets
+
+## **Setup and Config**
+
+### **Global config**
+
+- Clone or download repository to local machine.
+
+  > `git clone https://github.com/Wisdom-Nji/Afkanerd-OpenWebSockets.git`
+
+- Install dependencies with npm.
+
+  > `npm install`
+
+- Grant execute permission to install script.
+
+  > `sudo chmod 777 install.sh`
+
+- Run install script to configure client.
+
+  > `sudo ./install.sh`
+
+### **Server Setup**
+
+- Switch to server folder.
+
+  > `cd server`
+
+- Start server script with pm2.
+
+  > `pm2 start main.js -n "name your server instance"`
+
+- To view server log run pm2 log "number assigned to server instance by pm2 here".
+
+  > `pm2 log 0`
+
+### **Client Setup**
+
+- Switch to client folder.
+
+  > `cd ../client`
+
+- Start client script with pm2.
```

Comment:
It's better if you include the documentation for pm2 and explain why a developer should use that, than writing the instructions here. pm2 isn't a direct dependency of the entire platform

Reply:
Documentation as in what pm2 is and how it works ?